index.js

```diff
@@ -246,7 +246,7 @@ function rateLimitRequestHandler (params, pluginComponent) {
     if (params.addHeaders[params.labels.rateRemaining]) { res.header(params.labels.rateRemaining, 0) }
     if (params.addHeaders[params.labels.rateReset]) { res.header(params.labels.rateReset, timeLeft) }
     if (params.addHeaders[params.labels.retryAfter]) {
-      const resetAfterTime = (params.enableDraftSpec) ? timeLeft : params.timeWindow
+      const resetAfterTime = (params.enableDraftSpec ? timeLeft : params.timeWindow / 1000 | 0)
```

Comment:
Maybe we should precalculate it and have a params.timeWindowsInS.

But my first try to precalculate it was not successful. So want to know if we should precalculate it.

Reply:
solved